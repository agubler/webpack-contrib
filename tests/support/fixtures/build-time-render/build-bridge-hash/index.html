<html>
	<head>
		<link href="main.css" rel="stylesheet">
	</head>
	<body>
		<div id="app"></div>
		<script type="text/javascript" src="bootstrap.abcdefghij0123456789.bundle.js"></script>
		<script type="text/javascript" src="main.0123456789abcdefghij.bundle.js"></script>
	</body>
</html>
