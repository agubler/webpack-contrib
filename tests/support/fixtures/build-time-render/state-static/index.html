<html>
	<head>
		<link rel="manifest" href="manifest.json" />
		<link href="main.css" rel="stylesheet">
		<script src="http://localhost:9999/main.js"></script>
		<link href="http://localhost:9999/main.css" rel="stylesheet">
		<title>Original Title</title>
	</head>
	<body>
		<div id="app"></div>
		<script type="text/javascript" src="runtime.js"></script><script type="text/javascript" src="main.js"></script>
	</body>
</html>
