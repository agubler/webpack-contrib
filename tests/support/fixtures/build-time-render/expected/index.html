<html>
	<head>
		<style>
.hello {
  background: red;
}
span {
  width: 100%;
}
</style>
	</head>
	<body>
		<div id="app"></div>
		<script>
	(function () {
		var paths = [""];
		var html = ["<div id=\"app\"><div class=\"hello\">Hello</div></div>"];
		var element = document.getElementById('app');
		var target;
		paths.some(function (path, i) {
			target = html[i];
			return (typeof path === 'string' && path === window.location.hash) || path && (typeof path === 'object' && path.match && new RegExp(path.match.join('|')).test(window.location.hash));
		});
		if (target && element) {
			var frag = document.createRange().createContextualFragment(target);
			element.parentNode.replaceChild(frag, element);
		}
	}())
</script><link rel="stylesheet" href="main.css" media="none" onload="if(media!='all')media='all'" /><script type="text/javascript" src="runtime.js"></script><script type="text/javascript" src="main.js"></script>
	</body>
</html>
